Orange/widgets/cpewidget.py

```diff
@@ -2,6 +2,9 @@
 
 from Orange.widgets import widget
 from neuropype.engine.common import warn_once
+from neuropype.engine.ports import ListPort
+
+NoneUIValue = ['(use default)', 'default', 'defaults', '(use defaults)']
```

Comment:
I would not include "default" or "defaults" here -- the reason is that these may be legitimate values that some current or future nodes may accept, and then this code will turn it into None, which is almost certainly unexpected (whereas (default) or (use default) would be quite unlikely to be actual values). I do like the multiple cases though, and I'd suggest including (default) as well.